ui/src/main/resources/static/faq/topic-browser/fetch-faq.html

```diff
@@ -0,0 +1,53 @@
+<div class="faq">
+
+    <p>To perform a fetch operation, follow these steps:</p>
+
+    <ul>
+        <li>
+            <p>
+                Start by making the <b>first fetch</b> to populate the columns and headers lists.
+                These lists are used by filters and the column selector.
+            </p>
+        </li>
+        <li>
+            <p>
+                Each <b>Fetch</b> operation retrieves a specified number of records called the "Limit"
+                <b>(the limit can be set in the <img src="/kafka-manager/faq/topic-browser/img/limit-input-item.png"/> input item)</b>.
+                If the topic has multiple partitions, KM automatically calculates the limit for each partition.
+                The sum of all partition limits equals the total limit you entered. The limits for each partition are <b>distributed evenly</b>.
+                For example, if the "Limit" is set to 100 and the topic has 16 partitions, each partition will have a limit of 6 records.
+                Therefore, after a fetch operation, KM will attempt to fetch 6 new records from each partition.
+                If a specific partition does not have 6 records, all the remaining records in that partition will be fetched,
```

Comment:
It is not clear to me... If a partition does not have 6 record, but 4 only, where KM would fetch rest of the 2 records from?

Reply:
"all the remaining records in that partition will be fetched" that means that only 4 records will be fetched.

Reply:
Aaa... I got it the wrong way. You mean, that we will get only 4 records in the end...